### ISSUE TITLE
Feature Request: Allow scanning full projects in addition to individual repository

### ISSUE THREAD

**Issue by destornillador** [julienduchesne/pull-request-reminder#36]
Hello,

First of all I want to congrats you for the project. I tried it and it is the best Open Source PR Reminder for BitBucket I have found so far.

**Description of the Feature**
Today the tool allows scan individual repositories that are specified in the configuration file. In my use case, I have almost 100s repository and it makes difficult the maintenance of the config file. We use to group repositories in Bitbucket Projects so using it instead of single repositories, we don't have to bother about the config file every time that we want a new repository to be scanned. 

Config file could be something like this:
```
{
	"teams": [{
		"name": "boo",
		"hosts": {
			"bitbucket": {
				"projects": [
					"org/projects/project_name"
				]
			}
		}
	}]
}
```

Thank you

**Comment by julienduchesne** [comment]
Indeed, you're right. We could actually use that ourselves. I'll definitely look into that!

**Comment by julienduchesne** [comment]
This has been released in 0.3.0. Let me know if you have any issues with the feature!
